### PR TITLE
{bio}[intel/2016b] skewer v0.2.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2026.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/530a7d9e5f071ac34b2e8373f2717bf1 for a full test report.

File: easybuild/easyconfigs/s/skewer/skewer-0.2.2-intel-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+easyblock = 'MakeCp'
+
+name = 'skewer'
+version = '0.2.2'
+
+homepage = 'https://github.com/relipmoc/skewer'
+description = """skewer implements the bit-masked k-difference matching algorithm dedicated to the task of adapter
```
Comment (rjeschmi) [inline easybuild/easyconfigs/s/skewer/skewer-0.2.2-intel-2016b.eb]: It is weird that this doesn't start with a capital letter.

Comment (boegel) [inline easybuild/easyconfigs/s/skewer/skewer-0.2.2-intel-2016b.eb]: Well, I can change that if you really care, but the official software name is 'skewer', so...

I grabbed this from the README at https://github.com/relipmoc/skewer

Comment (rjeschmi) [inline easybuild/easyconfigs/s/skewer/skewer-0.2.2-intel-2016b.eb]: Yeah, they have a bold around the softwarename which makes it a bit clearer, but whatever, it doesn't really matter.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2575.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8b784758ce9fc025a0b2bc5dab6eb8b6 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @rjeschmi!

**Pull request merged**